master/getting-started/docker/tutorials/security-using-calico-profiles.md

```diff
@@ -1,8 +1,21 @@
 ---
-title: Simple Policy
+title: Security using Calico Profiles
 ---
 
-The following guide walks through a simple policy demo.
+## Background
+
+With Calico as a Docker network plugin, Calico uses an identically named
```

Comment:
Food for thought, a lot of the stuff mentioned in this paragraph can easily be "proven" showing sample output. I wonder if in the future we should rework this guide to focus on how this stuff works from a user perspective, rather than (what this guide seems to focus on which is) cross-host connectivity. 

No action here, just thinking aloud.